tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
-envlist = py27
-
-[testenv:py27]
-deps =
-    mock
-    mozfile >= 1.1
-    mozprocess >= 0.22
-    pylama
-    pytest
-    wptserve >= 1.4.0
+envlist = py27, coverage
 
+[testenv]
+deps = -rrequirements/tests.txt
 commands =
     pylama mozdownload
-    py.test {posargs}
```

Comment:
Perhaps you could leave the test environment as `py27` in anticipation for one day supporting Python 3.x too.